06-hot-reloading-with-directives/services/inventory/schema.js

```diff
@@ -0,0 +1,40 @@
+const { makeExecutableSchema } = require('@graphql-tools/schema');
+const { stitchingDirectives } = require('@graphql-tools/stitching-directives');
+
+const NotFoundError = require('../../lib/not_found_error');
+const readFileSync = require('../../lib/read_file_sync');
+
+const { stitchingDirectivesTypeDefs, stitchingDirectivesValidator } = stitchingDirectives();
```

Comment:
Why is `stitchingDirectives()` invoked?

Reply:
This gives an opportunity if desired to override the directive names, following convention of other GraphQL tools directives. This example is a little bit different because multiple directives are exported, but in that sense follows the same pattern.

Reply:
Ahhh, neat!